docs/route-tasks.md

```diff
@@ -0,0 +1,138 @@
+---
+id: route-tasks
+title: How to route Tasks
+sidebar_label: Route Tasks
+---
+
+Consider the specifications of a Worker:
+- A Worker is meant to handle only the Tasks that it is registered to handle. This means that Workers registered to handle Tasks for Workflow A are not capable of handling Tasks for Workflow B, and vice versa. 
+- A Worker runs on a speicific host. This means that Activity Tasks that depend on resources specific to a particular host must be executed by a Worker on that host.
+
+So, for any given Temporal Workflow, how do we make sure a specific Worker is executing a specific Task? 
+
+## Use Task Queues
+
+"Task Routing" is the development technique that makes use of Task Queues to orchestrate the execution of Tasks by specific Workers.
```

Comment:
This sentence is very wordy and sort of cyclical.

Reply:
Specifically " is the development technique that makes use of Task Queues" but the whole thing could probably be simpler.